src/types/types.ts

```diff
@@ -885,3 +885,123 @@ export interface PoiType {
 }
 
 export type PoiHiearachy = PoiType[]
+
+
+export interface Location extends LatLngId {
+  properties?: {
+    /** Parameter used for gravitational criteria to set a custom attraction strength for the given location
+     * @default 1
+     */
+    gravitationAttractionStrength: number
+  }
+}
+
+
+export interface PoiGroup {
+  /** key should always be 'group' */
+  key: 'group'
+
+  /** represents the id of the group */
+  value: string
+}
+
+/** See https://www.targomo.com/developers/resources/coverage/ for more infos regarding Service Coverage Areas*/
+export type CoreServiceUrl = (
+  'https://api.targomo.com/africa/' | 'https://api.targomo.com/asia/' | 'https://api.targomo.com/australia/' |
+  'https://api.targomo.com/britishisles/' | 'https://api.targomo.com/central_america/' | 'https://api.targomo.com/easterneurope/' |
+  'https://api.targomo.com/northamerica/' | 'https://api.targomo.com/south_america/' | 'https://api.targomo.com/westcentraleurope/')
+export type EndPoint = CoreServiceUrl
```

Comment:
I think there are even more possible url. Some of our customers have their own core service deployment. Maybe use a simple string for this field. It doesn't have to be checked out in the client, the API(s) will take care of it.